test/unit/puppetlabs/services/jruby/jruby_service_test.clj

```diff
@@ -31,7 +62,7 @@
        (try
          (bootstrap/with-app-with-config
           app
-          jruby-testutils/default-services
+          (conj jruby-testutils/default-services jruby-pooled-test-service)
```

Comment:
This test is testing that if an error happens during init, that the tk app will shut down. Since the `pool-manager-service` doesn't have anything happen during the init, it seemed like the best way to have this test was to include a service that has an `init` that is constructed in the way we expect users of this library to do. But maybe this test shouldn't exist at all anymore?